x/ccv/provider/keeper/grpc_query.go

```diff
@@ -105,39 +107,34 @@ func (k Keeper) GetConsumerChain(ctx sdk.Context, consumerId string) (types.Chai
 	}, nil
 }
 
-// TODO (PERMISSIONLESS)
 func (k Keeper) QueryConsumerChainStarts(goCtx context.Context, req *types.QueryConsumerChainStartProposalsRequest) (*types.QueryConsumerChainStartProposalsResponse, error) {
```

Comment:
Remove instead of deprecating. It will be a CLI breaking change, but we have a lot of them.

Reply:
See my comment [here](https://github.com/cosmos/interchain-security/pull/2177/files#r1734382529).